[01Jun2025 00:05:02.609] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! 1.5250102240740944,0.0,10.287433347119759
[01Jun2025 00:05:02.612] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! 1.583664463460991,0.0,10.683103860470624
[01Jun2025 00:05:02.612] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! 1.6423187028478878,0.0,11.078774373821489
[01Jun2025 00:05:02.612] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! 1.7009729422347846,0.0,11.474444887172353
[01Jun2025 00:05:02.612] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! 1.7596271816216813,0.0,11.870115400523218
[01Jun2025 00:05:02.612] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! 1.818281421008578,0.0,12.265785913874083
[01Jun2025 00:05:02.612] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! 1.8769356603954748,0.0,12.661456427224948
[01Jun2025 00:05:02.613] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! 1.9355898997823715,0.0,13.057126940575813
[01Jun2025 00:05:02.613] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! 1.9942441391692682,0.0,13.452797453926678
[01Jun2025 00:05:02.613] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! 2.052898378556165,0.0,13.848467967277543
[01Jun2025 00:05:02.613] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! 2.1115526179430617,0.0,14.244138480628408
[01Jun2025 00:05:02.613] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! 2.1702068573299584,0.0,14.639808993979273
[01Jun2025 00:05:02.613] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! 2.228861096716855,0.0,15.035479507330137
[01Jun2025 00:05:02.614] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! 2.287515336103752,0.0,15.431150020681002
[01Jun2025 00:05:02.614] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! 2.3461695754906486,0.0,15.826820534031867
[01Jun2025 00:05:02.614] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! 2.4048238148775454,0.0,16.222491047382732
[01Jun2025 00:05:02.614] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! 2.463478054264442,0.0,16.618161560733597
[01Jun2025 00:05:02.614] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! 2.522132293651339,0.0,17.013832074084462
[01Jun2025 00:05:02.615] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! 2.5807865330382356,0.0,17.409502587435327
[01Jun2025 00:05:02.615] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! 2.6394407724251323,0.0,17.80517310078619
[01Jun2025 00:05:02.615] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! 2.698095011812029,0.0,18.200843614137057
[01Jun2025 00:05:02.615] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! 2.7567492511989258,0.0,18.59651412748792
[01Jun2025 00:05:03.179] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.9355898997675922,0.0,-13.057126940578542
[01Jun2025 00:05:03.179] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.9355898997675922,0.0,-13.057126940578542
[01Jun2025 00:05:03.180] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.9355898997675922,0.0,-13.057126940578542
[01Jun2025 00:05:03.180] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.9355898997675922,0.0,-13.057126940578542
[01Jun2025 00:05:03.180] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.9355898997675922,0.0,-13.057126940578542
[01Jun2025 00:05:03.180] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.9355898997675922,0.0,-13.057126940578542
[01Jun2025 00:05:03.180] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.9355898997675922,0.0,-13.057126940578542
[01Jun2025 00:05:03.180] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.9355898997675922,0.0,-13.057126940578542
[01Jun2025 00:05:03.181] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.9355898997675922,0.0,-13.057126940578542
[01Jun2025 00:05:03.181] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.9355898997675922,0.0,-13.057126940578542
[01Jun2025 00:05:03.181] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.9355898997675922,0.0,-13.057126940578542
[01Jun2025 00:05:03.181] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.9355898997675922,0.0,-13.057126940578542
[01Jun2025 00:05:03.181] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.9355898997675922,0.0,-13.057126940578542
[01Jun2025 00:05:03.181] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.9355898997675922,0.0,-13.057126940578542
[01Jun2025 00:05:03.182] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.9355898997675922,0.0,-13.057126940578542
[01Jun2025 00:05:03.182] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.9355898997675922,0.0,-13.057126940578542
[01Jun2025 00:05:03.182] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.9355898997675922,0.0,-13.057126940578542
[01Jun2025 00:05:03.182] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.9355898997675922,0.0,-13.057126940578542
[01Jun2025 00:05:03.182] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.9355898997675922,0.0,-13.057126940578542
[01Jun2025 00:05:03.182] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.9355898997675922,0.0,-13.057126940578542
[01Jun2025 00:05:03.182] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.9355898997675922,0.0,-13.057126940578542
[01Jun2025 00:05:03.182] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.9355898997675922,0.0,-13.057126940578542
[01Jun2025 00:05:03.232] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.8769356603806955,0.0,-12.661456427227677
[01Jun2025 00:05:03.286] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.8182814209937987,0.0,-12.265785913876812
[01Jun2025 00:05:03.287] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.759627181606902,0.0,-11.870115400525947
[01Jun2025 00:05:03.384] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.7009729422200053,0.0,-11.474444887175082
[01Jun2025 00:05:03.396] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.6423187028331085,0.0,-11.078774373824217
[01Jun2025 00:05:03.480] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.5836644634462118,0.0,-10.683103860473352
[01Jun2025 00:05:03.532] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! -1.525010224059315,0.0,-10.287433347122487
[01Jun2025 00:05:03.639] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! 1.525010224059315,0.0,10.287433347122487
[01Jun2025 00:05:03.639] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of lautiABC) moved too quickly! 1.5836644634462118,0.0,10.683103860473352
[01Jun2025 00:06:46.901] [Server thread/INFO] [net.minecraft.server.players.PlayerList/]: YOUNGDINO07[/127.124.104.207:57125] logged in with entity id 24803 at (-4479.5, 61.6875, 120.5)
[01Jun2025 00:06:46.914] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: YOUNGDINO07 joined the game
[01Jun2025 00:06:46.928] [Server thread/INFO] [Jade/]: Syncing config to YOUNGDINO07 (ed45ce55-0981-3ac0-876f-3677ed698d8a)
[01Jun2025 00:06:48.547] [Server thread/INFO] [voicechat/]: [voicechat] Received secret request of YOUNGDINO07 (18)
[01Jun2025 00:06:48.548] [Server thread/INFO] [voicechat/]: [voicechat] Sent secret to YOUNGDINO07
[01Jun2025 00:06:54.911] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Successfully authenticated player ed45ce55-0981-3ac0-876f-3677ed698d8a
[01Jun2025 00:06:55.914] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Successfully validated connection of player ed45ce55-0981-3ac0-876f-3677ed698d8a
[01Jun2025 00:06:55.914] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Player YOUNGDINO07 (ed45ce55-0981-3ac0-876f-3677ed698d8a) successfully connected to voice chat
[01Jun2025 00:07:04.716] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lautiABC> agus crea grupo
[01Jun2025 00:07:08.720] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> oka
[01Jun2025 00:12:15.733] [Server thread/WARN] [net.minecraft.world.level.storage.loot.functions.SetItemDamageFunction/]: Couldn't set damage of loot item 0 air
[01Jun2025 00:12:32.755] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: YOUNGDINO07 was slain by Restless Spirit
[01Jun2025 00:13:19.237] [Worker-Main-2/INFO] [ModernFix/]: Instantiating Mojang DFU
[01Jun2025 00:15:45.545] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Player 1f8cbd0f-e34f-3631-ad78-ab523f2938ae timed out
[01Jun2025 00:15:45.545] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Reconnecting player Santix15
[01Jun2025 00:15:45.546] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Sent secret to Santix15
[01Jun2025 00:16:23.690] [Server thread/INFO] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Santix15 lost connection: Disconnected
[01Jun2025 00:16:23.690] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Santix15 left the game
[01Jun2025 00:16:23.693] [Server thread/INFO] [voicechat/]: [voicechat] Disconnecting client Santix15
[01Jun2025 00:16:23.741] [Server thread/WARN] [net.minecraft.network.Connection/]: handleDisconnection() called twice
[01Jun2025 00:17:38.168] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 00:17:38.169] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=1000, z=1073200}"
}
[01Jun2025 00:17:38.169] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 00:17:38.169] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[108, -88, 37, -60, -88, -56, -61, -74, -24, 94, 9]"
]
[01Jun2025 00:17:39.295] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 00:17:39.296] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=791, z=1073200}"
}
[01Jun2025 00:17:39.296] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 00:17:39.296] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[12, -3, 61, 90, 1, 23, -106, 104, 23, 46, -47]"
]
[01Jun2025 00:17:40.610] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 00:17:40.610] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=916, z=1073200}"
}
[01Jun2025 00:17:40.610] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 00:17:40.611] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[-21, -62, 101, -1, -124, -23, -38, 21, 89, 47, -79]"
]
[01Jun2025 00:17:40.947] [Netty Epoll Server IO #0/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 00:17:40.947] [Netty Epoll Server IO #0/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=780, z=1073200}"
}
[01Jun2025 00:17:40.947] [Netty Epoll Server IO #0/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 00:17:40.947] [Netty Epoll Server IO #0/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[119, -75, -9, 2, 28, 36, -28, 51, 47, -63, -111]"
]
[01Jun2025 00:17:41.739] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 00:17:41.739] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=801, z=1073200}"
}
[01Jun2025 00:17:41.739] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 00:17:41.740] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[-4, -10, -88, -123, 124, -120, 89, -33, -84, 103, -4]"
]
[01Jun2025 00:17:41.968] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 00:17:41.969] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=868, z=1073200}"
}
[01Jun2025 00:17:41.970] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 00:17:41.970] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[-23, -35, 0, 59, -64, 93, 51, -114, 25, 12, 16]"
]
[01Jun2025 00:17:42.076] [Netty Epoll Server IO #3/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 00:17:42.076] [Netty Epoll Server IO #3/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=806, z=1073200}"
}
[01Jun2025 00:17:42.077] [Netty Epoll Server IO #3/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 00:17:42.077] [Netty Epoll Server IO #3/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[60, -49, -42, -100, 122, -94, 9, 114, -7, -66, 33]"
]
[01Jun2025 00:17:42.195] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 00:17:42.195] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=5768192, y=953, z=1122352}"
}
[01Jun2025 00:17:42.195] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 00:17:42.195] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "$ma(WkBF"
]
[01Jun2025 00:17:43.859] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 00:17:43.859] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 0,
  "f_133799_": "BlockPos{x=-8125440, y=835, z=905264}"
}
[01Jun2025 00:17:43.859] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 00:17:43.859] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[70, -26, 11, -119, 35, 10, 48, 73, 6, -104, 29]"
]
[01Jun2025 00:17:43.997] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 00:17:43.998] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=964, z=1073200}"
}
[01Jun2025 00:17:43.998] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 00:17:43.998] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[124, 55, -79, 76, 88, -119, -37, 77, 22, 93, 108]"
]
[01Jun2025 00:17:44.399] [Netty Epoll Server IO #0/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 00:17:44.399] [Netty Epoll Server IO #0/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=1023, z=1073200}"
}
[01Jun2025 00:17:44.400] [Netty Epoll Server IO #0/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 00:17:44.400] [Netty Epoll Server IO #0/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "d!qK\"$sA"
]
[01Jun2025 00:17:53.009] [Server thread/INFO] [net.minecraft.server.players.PlayerList/]: Santix15[/127.71.247.24:51507] logged in with entity id 33102 at (-1138.5141776130681, -55.0, 2607.949647880651)
[01Jun2025 00:17:53.037] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Santix15 joined the game
[01Jun2025 00:17:53.057] [Server thread/INFO] [Jade/]: Syncing config to Santix15 (1f8cbd0f-e34f-3631-ad78-ab523f2938ae)
[01Jun2025 00:17:54.709] [Server thread/INFO] [voicechat/]: [voicechat] Received secret request of Santix15 (18)
[01Jun2025 00:17:54.709] [Server thread/INFO] [voicechat/]: [voicechat] Sent secret to Santix15
[01Jun2025 00:18:03.883] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Successfully authenticated player 1f8cbd0f-e34f-3631-ad78-ab523f2938ae
[01Jun2025 00:18:05.026] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Successfully validated connection of player 1f8cbd0f-e34f-3631-ad78-ab523f2938ae
[01Jun2025 00:18:05.027] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Player Santix15 (1f8cbd0f-e34f-3631-ad78-ab523f2938ae) successfully connected to voice chat
[01Jun2025 00:19:27.690] [Server thread/WARN] [net.minecraft.server.MinecraftServer/]: Can't keep up! Is the server overloaded? Running 2025ms or 40 ticks behind
[01Jun2025 00:19:45.556] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lobo3224> holaaa santi
[01Jun2025 00:21:20.303] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: lautiABC has made the advancement [Best Friends Forever]
[01Jun2025 00:22:10.030] [Worker-Main-79/WARN] [net.minecraft.world.entity.ai.attributes.AttributeMap/]: Ignoring unknown attribute 'reach-entity-attributes:reach'
[01Jun2025 00:22:10.031] [Worker-Main-79/WARN] [net.minecraft.world.entity.ai.attributes.AttributeMap/]: Ignoring unknown attribute 'reach-entity-attributes:reach'
[01Jun2025 00:22:57.400] [Worker-Main-2/WARN] [net.minecraft.world.entity.ai.attributes.AttributeMap/]: Ignoring unknown attribute 'reach-entity-attributes:reach'
[01Jun2025 00:22:57.401] [Worker-Main-2/WARN] [net.minecraft.world.entity.ai.attributes.AttributeMap/]: Ignoring unknown attribute 'reach-entity-attributes:reach'
[01Jun2025 00:25:48.647] [Server thread/WARN] [net.minecraft.world.level.storage.loot.functions.SetItemDamageFunction/]: Couldn't set damage of loot item 0 air
[01Jun2025 00:29:44.396] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundAcceptTeleportationPacket  id 0 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 00:29:44.396] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133785_": 81
}
[01Jun2025 00:29:44.397] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 0 data of 48 extra bytes: 
[01Jun2025 00:29:44.397] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "2+GbZ\"\\)DL:z*320\"%!]O\"wp"
]
[01Jun2025 00:29:59.143] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of YOUNGDINO07) moved wrongly! 0.6905476272891065
[01Jun2025 00:29:59.199] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of YOUNGDINO07) moved wrongly! 0.8408204509521511
[01Jun2025 00:29:59.255] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of YOUNGDINO07) moved wrongly! 0.8795680074597385
[01Jun2025 00:29:59.329] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of YOUNGDINO07) moved wrongly! 0.9531883639003809
[01Jun2025 00:29:59.330] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of YOUNGDINO07) moved wrongly! 1.0581942394492216
[01Jun2025 00:29:59.413] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of YOUNGDINO07) moved wrongly! 1.1914470814472224
[01Jun2025 00:29:59.457] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of YOUNGDINO07) moved wrongly! 1.3501221925760092
[01Jun2025 00:29:59.509] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of YOUNGDINO07) moved wrongly! 1.5316773453164387
[01Jun2025 00:29:59.604] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of YOUNGDINO07) moved wrongly! 0.32336188452973147
[01Jun2025 00:31:52.547] [Worker-Main-12/WARN] [ModernFix/]: Skipped emitting ENTITY_MOUNT game event for entity Strider['Strider'/49134, l='ServerLevel[world]', x=-578.00, y=31.00, z=-507.00] as it would cause deadlock
[01Jun2025 00:32:28.253] [Worker-Main-2/WARN] [ModernFix/]: Skipped emitting ENTITY_MOUNT game event for entity Strider['Strider'/49502, l='ServerLevel[world]', x=-525.00, y=31.00, z=-555.00] as it would cause deadlock
[01Jun2025 00:32:40.614] [Worker-Main-11/ERROR] [net.minecraft.world.entity.decoration.HangingEntity/]: Hanging entity at invalid position: BlockPos{x=8, y=-59, z=111}
[01Jun2025 00:32:40.615] [Worker-Main-11/ERROR] [net.minecraft.world.entity.decoration.HangingEntity/]: Hanging entity at invalid position: BlockPos{x=8, y=-59, z=115}
[01Jun2025 00:32:40.615] [Worker-Main-11/ERROR] [net.minecraft.world.entity.decoration.HangingEntity/]: Hanging entity at invalid position: BlockPos{x=8, y=-59, z=119}
[01Jun2025 00:32:40.615] [Worker-Main-11/ERROR] [net.minecraft.world.entity.decoration.HangingEntity/]: Hanging entity at invalid position: BlockPos{x=8, y=-59, z=123}
[01Jun2025 00:32:40.620] [Worker-Main-2/ERROR] [net.minecraft.world.entity.decoration.HangingEntity/]: Hanging entity at invalid position: BlockPos{x=8, y=-59, z=95}
[01Jun2025 00:32:40.620] [Worker-Main-2/ERROR] [net.minecraft.world.entity.decoration.HangingEntity/]: Hanging entity at invalid position: BlockPos{x=8, y=-59, z=99}
[01Jun2025 00:32:40.620] [Worker-Main-2/ERROR] [net.minecraft.world.entity.decoration.HangingEntity/]: Hanging entity at invalid position: BlockPos{x=8, y=-59, z=103}
[01Jun2025 00:32:40.620] [Worker-Main-2/ERROR] [net.minecraft.world.entity.decoration.HangingEntity/]: Hanging entity at invalid position: BlockPos{x=8, y=-59, z=107}
[01Jun2025 00:32:45.591] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: lobo3224 has made the advancement [A Terrible Fortress]
[01Jun2025 00:32:47.015] [Worker-Main-2/WARN] [ModernFix/]: Skipped emitting ENTITY_MOUNT game event for entity Strider['Strider'/49984, l='ServerLevel[world]', x=-520.00, y=31.00, z=-565.00] as it would cause deadlock
[01Jun2025 00:33:15.367] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: lobo3224 has made the advancement [Frightening Rush]
[01Jun2025 00:33:18.455] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: lobo3224 has made the advancement [Phasmophobic]
[01Jun2025 00:33:38.810] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundAcceptTeleportationPacket  id 0 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 00:33:38.858] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133785_": 28136073
}
[01Jun2025 00:33:38.859] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 0 data of 45 extra bytes: 
[01Jun2025 00:33:38.859] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "2+v3pWA320X<*J"
]
[01Jun2025 00:33:48.389] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: YOUNGDINO07 was killed
[01Jun2025 00:33:51.952] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lobo3224> xd
[01Jun2025 00:34:25.900] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: YOUNGDINO07 was killed
[01Jun2025 00:35:09.708] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of YOUNGDINO07) moved wrongly! 0.42825287457287686
[01Jun2025 00:35:09.755] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of YOUNGDINO07) moved wrongly! 0.7636912732405108
[01Jun2025 00:35:09.805] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of YOUNGDINO07) moved wrongly! 1.0964599670745656
[01Jun2025 00:35:09.847] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of YOUNGDINO07) moved wrongly! 1.4268234864961187
[01Jun2025 00:35:09.899] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of YOUNGDINO07) moved wrongly! 1.7550199107465687
[01Jun2025 00:35:09.948] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of YOUNGDINO07) moved wrongly! 2.0812635092803475
[01Jun2025 00:35:10.054] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of YOUNGDINO07) moved wrongly! 0.2523130062598966
[01Jun2025 00:35:10.054] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Boat (vehicle of YOUNGDINO07) moved wrongly! 0.5617027423991203
[01Jun2025 00:37:18.813] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -30], pos: BlockPos{x=-688, y=107, z=-465}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.813] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -30], pos: BlockPos{x=-688, y=106, z=-465}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.814] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=107, z=-464}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.814] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=106, z=-464}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.814] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=107, z=-463}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.814] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=106, z=-463}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.814] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=107, z=-462}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.814] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=106, z=-462}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.815] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=107, z=-461}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.815] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=106, z=-461}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.816] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=107, z=-457}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.816] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=106, z=-457}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.816] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=107, z=-456}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.816] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=106, z=-456}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.816] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=107, z=-455}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.816] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=106, z=-455}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.816] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=107, z=-454}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.816] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -30], pos: BlockPos{x=-687, y=106, z=-465}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.816] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -30], pos: BlockPos{x=-688, y=105, z=-465}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.816] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=107, z=-464}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.817] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=106, z=-464}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.817] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=105, z=-464}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.817] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=107, z=-463}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.817] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=106, z=-463}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.817] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=105, z=-463}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.817] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=107, z=-462}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.817] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=106, z=-462}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.817] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=105, z=-462}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.817] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=107, z=-461}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.817] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=106, z=-461}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.817] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=105, z=-461}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.817] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=107, z=-459}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.817] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=106, z=-459}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.818] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=107, z=-458}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.818] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=106, z=-458}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.818] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=106, z=-457}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.818] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=105, z=-457}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.818] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=106, z=-456}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.818] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=105, z=-456}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.818] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=107, z=-455}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.818] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=106, z=-455}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.818] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=105, z=-455}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.818] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=106, z=-464}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.818] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=105, z=-464}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.822] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=104, z=-464}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.822] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=103, z=-464}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.822] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=107, z=-463}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.822] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=106, z=-463}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.823] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=105, z=-463}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.823] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=104, z=-463}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.823] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=103, z=-463}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.823] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=107, z=-462}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.823] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=106, z=-462}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.823] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=105, z=-462}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.823] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=104, z=-462}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.823] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=103, z=-462}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.823] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=105, z=-461}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.824] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=104, z=-461}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.824] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=103, z=-461}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.824] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=107, z=-460}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.824] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=106, z=-460}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.824] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=105, z=-460}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.824] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=103, z=-460}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.825] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=105, z=-459}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.825] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=104, z=-459}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.825] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=103, z=-459}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.825] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=107, z=-458}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.825] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=106, z=-458}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.825] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=105, z=-458}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.825] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=103, z=-458}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.826] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=107, z=-457}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.826] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=106, z=-457}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.827] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=105, z=-457}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.827] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=104, z=-457}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.828] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=103, z=-457}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.828] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=107, z=-456}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.828] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=106, z=-456}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.829] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=105, z=-456}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.829] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=104, z=-456}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.829] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=103, z=-456}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.830] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=107, z=-455}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.830] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=106, z=-455}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.830] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-685, y=106, z=-464}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.831] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=105, z=-464}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.831] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=105, z=-463}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.831] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=104, z=-463}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.832] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=103, z=-463}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.832] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=102, z=-463}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.833] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=105, z=-462}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.833] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=104, z=-462}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.833] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=103, z=-462}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.834] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=102, z=-462}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.834] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=105, z=-461}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.835] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=104, z=-461}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.837] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=103, z=-461}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.837] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=102, z=-461}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.839] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-685, y=107, z=-460}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.839] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-685, y=106, z=-460}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.840] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=105, z=-460}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.840] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=104, z=-460}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.840] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=103, z=-460}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.841] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=102, z=-460}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.841] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-685, y=106, z=-459}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.842] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=105, z=-459}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.842] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=104, z=-459}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.842] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=103, z=-459}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.842] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=102, z=-459}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.842] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-685, y=107, z=-458}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.843] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-685, y=106, z=-458}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.843] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=105, z=-458}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.843] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=104, z=-458}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.843] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=103, z=-458}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.843] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=102, z=-458}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.843] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-685, y=107, z=-457}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.843] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-685, y=106, z=-457}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.843] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=105, z=-457}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.843] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=104, z=-457}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.843] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=103, z=-457}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.843] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-688, y=102, z=-457}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.843] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-685, y=107, z=-456}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.844] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-685, y=106, z=-456}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.844] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=105, z=-456}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.845] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-685, y=107, z=-455}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.845] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-684, y=106, z=-463}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.846] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-685, y=105, z=-463}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.846] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-684, y=106, z=-462}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.846] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=104, z=-462}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.846] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=103, z=-462}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.846] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-684, y=106, z=-461}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.846] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=104, z=-461}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.847] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=103, z=-461}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.847] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=102, z=-461}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.848] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-684, y=107, z=-460}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.848] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-684, y=106, z=-460}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.848] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-685, y=105, z=-460}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.848] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=104, z=-460}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.849] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=103, z=-460}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.849] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-687, y=102, z=-460}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.849] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-685, y=105, z=-459}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.849] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=104, z=-459}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.849] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=103, z=-459}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.849] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-684, y=106, z=-458}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.849] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-685, y=105, z=-458}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.849] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-686, y=104, z=-458}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.850] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-685, y=105, z=-457}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.850] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-684, y=107, z=-456}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.850] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-683, y=106, z=-462}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.850] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-683, y=106, z=-461}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.850] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-684, y=105, z=-461}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.850] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-683, y=106, z=-460}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.850] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-684, y=105, z=-460}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.850] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-683, y=106, z=-458}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:37:18.850] [Worker-Main-11/ERROR] [net.minecraft.Util/]: Detected setBlock in a far chunk [-43, -29], pos: BlockPos{x=-683, y=107, z=-457}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/placed_feature / netherexp:soul_sand_valley/black_ice_glaciers/large_black_ice]
[01Jun2025 00:38:37.965] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Player 71cf154b-6e54-326e-9700-98d60017873d timed out
[01Jun2025 00:38:37.966] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Reconnecting player lautiABC
[01Jun2025 00:38:37.966] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Sent secret to lautiABC
[01Jun2025 00:38:47.329] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Successfully authenticated player 71cf154b-6e54-326e-9700-98d60017873d
[01Jun2025 00:38:48.409] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Successfully validated connection of player 71cf154b-6e54-326e-9700-98d60017873d
[01Jun2025 00:38:48.409] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Player lautiABC (71cf154b-6e54-326e-9700-98d60017873d) successfully connected to voice chat
[01Jun2025 00:38:58.591] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: lobo3224 has reached the goal [The Fright Before Xmas]
[01Jun2025 00:39:20.093] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: YOUNGDINO07 was slain by Deepling Brute
[01Jun2025 00:39:56.541] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: YOUNGDINO07 was slain by Deepling Brute
[01Jun2025 00:40:13.600] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: lobo3224 has completed the challenge [Hot Tourist Destinations]
[01Jun2025 00:40:22.622] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: YOUNGDINO07 was slain by Deepling Brute
[01Jun2025 00:40:31.636] [Worker-Main-11/WARN] [ModernFix/]: Skipped emitting ENTITY_MOUNT game event for entity Strider['Strider'/58771, l='ServerLevel[world]', x=-1026.00, y=31.00, z=-429.00] as it would cause deadlock
[01Jun2025 00:41:20.237] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [Santix15: Teleported lobo3224 to Santix15]
[01Jun2025 00:45:12.303] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lautiABC> gonza el agus dice gonza q gusano de mierda hijo de tremil puta xd lol
[01Jun2025 00:45:19.990] [Server thread/WARN] [net.minecraft.world.level.storage.loot.functions.SetItemDamageFunction/]: Couldn't set damage of loot item 0 air
[01Jun2025 00:45:22.233] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lobo3224> XD
[01Jun2025 00:45:59.336] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: lobo3224 has made the advancement [Bypassing the Expiration Date]
[01Jun2025 00:46:11.382] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Santix15 has made the advancement [Bypassing the Expiration Date]
[01Jun2025 00:48:51.041] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: YOUNGDINO07 was killed
[01Jun2025 00:49:43.148] [Server thread/WARN] [net.minecraft.server.level.ServerPlayerGameMode/]: Mismatch in destroy block pos: BlockPos{x=-134, y=12, z=342} BlockPos{x=-135, y=12, z=342}
[01Jun2025 00:51:13.740] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundAcceptTeleportationPacket  id 0 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 00:51:13.741] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133785_": 1869581
}
[01Jun2025 00:51:13.743] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 0 data of 46 extra bytes: 
[01Jun2025 00:51:13.743] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "9~Y_tT^>%3525ki >"
]
[01Jun2025 00:55:06.140] [Server thread/INFO] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: lautiABC lost connection: Disconnected
[01Jun2025 00:55:06.141] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: lautiABC left the game
[01Jun2025 00:55:06.141] [Server thread/INFO] [voicechat/]: [voicechat] Disconnecting client lautiABC
[01Jun2025 00:55:54.329] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundAcceptTeleportationPacket  id 0 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 00:55:54.330] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133785_": 15612
}
[01Jun2025 00:55:54.330] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 0 data of 47 extra bytes: 
[01Jun2025 00:55:54.330] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "q&0-95>+N320/L?2"
]
[01Jun2025 00:59:35.046] [Server thread/INFO] [net.minecraft.server.players.PlayerList/]: lautiABC[/127.52.80.31:58925] logged in with entity id 74212 at (-1496.5, 81.0, 2191.5)
[01Jun2025 00:59:35.057] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: lautiABC joined the game
[01Jun2025 00:59:35.080] [Server thread/INFO] [Jade/]: Syncing config to lautiABC (71cf154b-6e54-326e-9700-98d60017873d)
[01Jun2025 00:59:36.440] [Server thread/INFO] [voicechat/]: [voicechat] Received secret request of lautiABC (18)
[01Jun2025 00:59:36.440] [Server thread/INFO] [voicechat/]: [voicechat] Sent secret to lautiABC
[01Jun2025 00:59:45.187] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Successfully authenticated player 71cf154b-6e54-326e-9700-98d60017873d
[01Jun2025 00:59:46.184] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Successfully validated connection of player 71cf154b-6e54-326e-9700-98d60017873d
[01Jun2025 00:59:46.185] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Player lautiABC (71cf154b-6e54-326e-9700-98d60017873d) successfully connected to voice chat
[01Jun2025 01:00:48.834] [Server thread/WARN] [net.minecraft.world.level.storage.loot.functions.SetItemDamageFunction/]: Couldn't set damage of loot item 0 air
[01Jun2025 01:07:26.878] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: lautiABC was frozen to death by Santix15
[01Jun2025 01:09:27.129] [Server thread/WARN] [net.minecraft.world.level.storage.loot.functions.SetItemDamageFunction/]: Couldn't set damage of loot item 0 air
[01Jun2025 01:09:37.727] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: lautiABC withered away whilst fighting lobo3224
[01Jun2025 01:09:39.928] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: YOUNGDINO07 was killed by magic whilst trying to escape lobo3224
[01Jun2025 01:10:05.578] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: lautiABC withered away whilst fighting lobo3224
[01Jun2025 01:10:14.576] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: YOUNGDINO07 froze to death
[01Jun2025 01:10:20.032] [Server thread/WARN] [net.minecraft.server.level.ServerPlayerGameMode/]: Mismatch in destroy block pos: BlockPos{x=0, y=0, z=0} BlockPos{x=54, y=73, z=30}
[01Jun2025 01:10:20.080] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: YOUNGDINO07 moved wrongly!
[01Jun2025 01:10:20.453] [Netty Epoll Server IO #6/ERROR] [net.minecraftforge.network.simple.IndexedMessageCodec/SIMPLENET]: Received empty payload on channel fml:handshake
[01Jun2025 01:10:20.460] [Netty Epoll Server IO #6/INFO] [net.minecraft.server.network.ServerLoginPacketListenerImpl/]: Disconnecting com.mojang.authlib.GameProfile@560b1328[id=<null>,name=maximito,properties={},legacy=false] (/127.141.4.49:59601): Unexpected custom data from client
[01Jun2025 01:10:20.484] [Server thread/INFO] [net.minecraft.server.network.ServerLoginPacketListenerImpl/]: com.mojang.authlib.GameProfile@560b1328[id=<null>,name=maximito,properties={},legacy=false] (/127.141.4.49:59601) lost connection: Unexpected custom data from client
[01Jun2025 01:10:31.436] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: lautiABC froze to death
[01Jun2025 01:10:56.630] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lobo3224> jjajajaja
[01Jun2025 01:11:13.561] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> con poderes cualquiera xd
[01Jun2025 01:11:14.480] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lobo3224> venganza
[01Jun2025 01:11:32.789] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lautiABC> pero no cantes victoria por ahora
[01Jun2025 01:11:37.735] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lobo3224> te mate con unaa espada
[01Jun2025 01:11:50.256] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lautiABC> esto resien enpieza gonza esto recien enpieza
[01Jun2025 01:12:38.664] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 01:12:38.665] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=971, z=1073200}"
}
[01Jun2025 01:12:38.665] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 01:12:38.665] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[117, -107, -48, -5, -37, -123, -12, 76, 46, -76, -68]"
]
[01Jun2025 01:12:40.094] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 01:12:40.095] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=886, z=1073200}"
}
[01Jun2025 01:12:40.095] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 01:12:40.095] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[42, -43, 86, 83, -128, -92, -59, -107, 69, 14, 36]"
]
[01Jun2025 01:12:41.146] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 01:12:41.146] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=925, z=1073200}"
}
[01Jun2025 01:12:41.147] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 01:12:41.147] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[109, -23, -23, 126, -106, 118, -85, -22, 25, -107, 93]"
]
[01Jun2025 01:12:41.525] [Netty Epoll Server IO #3/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 01:12:41.525] [Netty Epoll Server IO #3/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=878, z=1073200}"
}
[01Jun2025 01:12:41.525] [Netty Epoll Server IO #3/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 01:12:41.526] [Netty Epoll Server IO #3/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[-75, 16, 90, -53, -94, -127, 2, -69, 20, 89, 31]"
]
[01Jun2025 01:12:42.274] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 01:12:42.274] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=772, z=1073200}"
}
[01Jun2025 01:12:42.275] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 01:12:42.275] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[28, -66, -4, 58, -116, -93, 39, 57, 123, 111, 52]"
]
[01Jun2025 01:12:45.548] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 01:12:45.549] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=782, z=1073200}"
}
[01Jun2025 01:12:45.549] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 01:12:45.549] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[-92, 20, -99, 57, -116, -28, 3, -38, -11, -118, -124]"
]
[01Jun2025 01:12:45.600] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 01:12:45.600] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=811, z=1073200}"
}
[01Jun2025 01:12:45.600] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 01:12:45.600] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[82, 40, -28, -52, -68, 30, -1, 118, 77, 65, 119]"
]
[01Jun2025 01:12:54.096] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 01:12:54.096] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=5768192, y=882, z=1122352}"
}
[01Jun2025 01:12:54.096] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 01:12:54.096] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[3, -12, -103, 100, -48, -91, -37, 75, 125, 84, 35]"
]
[01Jun2025 01:12:54.642] [Netty Epoll Server IO #0/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 01:12:54.642] [Netty Epoll Server IO #0/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 0,
  "f_133799_": "BlockPos{x=-8125440, y=776, z=905264}"
}
[01Jun2025 01:12:54.642] [Netty Epoll Server IO #0/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 01:12:54.642] [Netty Epoll Server IO #0/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[-22, -32, 81, -38, 5, 24, 108, 108, 37, -27, -110]"
]
[01Jun2025 01:12:54.914] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 01:12:54.914] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 0,
  "f_133799_": "BlockPos{x=-11271168, y=945, z=856112}"
}
[01Jun2025 01:12:54.914] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 01:12:54.914] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[-10, -88, -44, -76, 59, 62, 123, 60, -2, 27, 107]"
]
[01Jun2025 01:14:05.040] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: YOUNGDINO07 tried to swim in lava
[01Jun2025 01:14:31.038] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: lautiABC was slain by Hollow Soldier
[01Jun2025 01:17:20.411] [Netty Epoll Server IO #4/ERROR] [net.minecraftforge.network.simple.IndexedMessageCodec/SIMPLENET]: Received empty payload on channel fml:handshake
[01Jun2025 01:17:20.411] [Netty Epoll Server IO #4/INFO] [net.minecraft.server.network.ServerLoginPacketListenerImpl/]: Disconnecting com.mojang.authlib.GameProfile@5cbb060d[id=<null>,name=maximito,properties={},legacy=false] (/127.141.4.49:57719): Unexpected custom data from client
[01Jun2025 01:17:20.433] [Server thread/INFO] [net.minecraft.server.network.ServerLoginPacketListenerImpl/]: com.mojang.authlib.GameProfile@5cbb060d[id=<null>,name=maximito,properties={},legacy=false] (/127.141.4.49:57719) lost connection: Unexpected custom data from client
[01Jun2025 01:18:01.049] [Netty Epoll Server IO #5/ERROR] [net.minecraftforge.network.simple.IndexedMessageCodec/SIMPLENET]: Received empty payload on channel fml:handshake
[01Jun2025 01:18:01.049] [Netty Epoll Server IO #5/INFO] [net.minecraft.server.network.ServerLoginPacketListenerImpl/]: Disconnecting com.mojang.authlib.GameProfile@4d6418b2[id=<null>,name=maximito,properties={},legacy=false] (/127.141.4.49:49403): Unexpected custom data from client
[01Jun2025 01:18:01.082] [Server thread/INFO] [net.minecraft.server.network.ServerLoginPacketListenerImpl/]: com.mojang.authlib.GameProfile@4d6418b2[id=<null>,name=maximito,properties={},legacy=false] (/127.141.4.49:49403) lost connection: Unexpected custom data from client
[01Jun2025 01:18:14.682] [Server thread/WARN] [net.minecraft.world.level.storage.loot.functions.SetItemDamageFunction/]: Couldn't set damage of loot item 0 air
[01Jun2025 01:22:27.031] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Santix15 has made the advancement [A Seedy Place]
[01Jun2025 01:23:09.380] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundAcceptTeleportationPacket  id 0 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 01:23:09.381] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133785_": 105
}
[01Jun2025 01:23:09.381] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 0 data of 48 extra bytes: 
[01Jun2025 01:23:09.381] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "i>D(Ldq<,320pvZD^"
]
[01Jun2025 01:24:47.188] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: lautiABC was killed by magic whilst trying to escape lobo3224
[01Jun2025 01:29:53.977] [Server thread/WARN] [net.minecraft.world.level.storage.loot.functions.SetItemDamageFunction/]: Couldn't set damage of loot item 0 air
[01Jun2025 01:29:53.977] [Server thread/WARN] [net.minecraft.world.level.storage.loot.functions.SetItemDamageFunction/]: Couldn't set damage of loot item 0 air
[01Jun2025 01:31:46.625] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: lautiABC was slain by Murmur
[01Jun2025 01:33:20.083] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lobo3224> vete
[01Jun2025 01:35:55.235] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: YOUNGDINO07 was slain by Zombie
[01Jun2025 01:36:03.586] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [Santix15: Set the weather to clear]
[01Jun2025 01:36:18.255] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [Santix15: Set the time to 1000]
[01Jun2025 01:36:24.890] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <Santix15> ayudita
[01Jun2025 01:36:42.616] [Server thread/INFO] [net.minecraft.server.players.PlayerList/]: kevedo909[/127.5.252.102:37765] logged in with entity id 98640 at (-4480.5, 61.6875, 125.5)
[01Jun2025 01:36:42.628] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: kevedo909 joined the game
[01Jun2025 01:36:42.648] [Server thread/INFO] [Jade/]: Syncing config to kevedo909 (380318f8-90c8-30a1-b2c2-428693a3e9ab)
[01Jun2025 01:37:00.743] [Server thread/INFO] [voicechat/]: [voicechat] Received secret request of kevedo909 (18)
[01Jun2025 01:37:00.744] [Server thread/INFO] [voicechat/]: [voicechat] Sent secret to kevedo909
[01Jun2025 01:37:02.098] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: kevedo909 suffocated in a wall
[01Jun2025 01:37:16.986] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> por gil
[01Jun2025 01:37:24.008] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> casi me muero yo igual tarado
[01Jun2025 01:37:38.846] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Successfully authenticated player 380318f8-90c8-30a1-b2c2-428693a3e9ab
[01Jun2025 01:37:39.897] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Successfully validated connection of player 380318f8-90c8-30a1-b2c2-428693a3e9ab
[01Jun2025 01:37:39.898] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Player kevedo909 (380318f8-90c8-30a1-b2c2-428693a3e9ab) successfully connected to voice chat
[01Jun2025 01:38:25.342] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <kevedo909> como mori
[01Jun2025 01:38:35.991] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <Santix15> por una pared
[01Jun2025 01:38:42.449] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <kevedo909> afixiado por una pared
[01Jun2025 01:38:42.540] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> te puse bloques de tierra en el spawn
[01Jun2025 01:38:44.056] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> xd
[01Jun2025 01:38:46.755] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <kevedo909> que mierda
[01Jun2025 01:38:50.114] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> vos igual lo hiciste
[01Jun2025 01:38:52.106] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <kevedo909> agus
[01Jun2025 01:38:53.939] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> dea
[01Jun2025 01:38:55.512] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> queu
[01Jun2025 01:39:03.496] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <kevedo909> sos idiota o te haces
[01Jun2025 01:39:10.343] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Santix15 moved wrongly!
[01Jun2025 01:39:10.762] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> x2
[01Jun2025 01:39:24.262] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <kevedo909> mas de 6000 bloques
[01Jun2025 01:39:33.694] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> mas de 70000 bloques
[01Jun2025 01:39:42.699] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <kevedo909> por eso estoy pensando de traicionarlos...
[01Jun2025 01:39:51.412] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> que te dije lauti
[01Jun2025 01:39:57.893] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> intentalo si queres
[01Jun2025 01:40:00.163] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> ahh
[01Jun2025 01:40:03.513] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lautiABC> jaajajajajaj
[01Jun2025 01:40:08.574] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> re asesino yo xd
[01Jun2025 01:40:17.407] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <kevedo909> porque mierda pusiste esa paredes
[01Jun2025 01:40:34.709] [Server thread/WARN] [net.minecraft.server.level.ServerPlayerGameMode/]: Mismatch in destroy block pos: BlockPos{x=0, y=0, z=0} BlockPos{x=45, y=66, z=30}
[01Jun2025 01:42:17.958] [Server thread/WARN] [net.minecraft.server.level.ServerPlayerGameMode/]: Mismatch in destroy block pos: BlockPos{x=-2348, y=64, z=2815} BlockPos{x=-2349, y=62, z=2814}
[01Jun2025 01:43:15.457] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <kevedo909> no se escucha nada
[01Jun2025 01:43:25.047] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <kevedo909> estan hablando?
[01Jun2025 01:43:26.860] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> nos salimos
[01Jun2025 01:43:28.753] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> no
[01Jun2025 01:43:35.912] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <kevedo909> jaja que chistodos
[01Jun2025 01:43:42.082] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> unite lauti
[01Jun2025 01:43:45.947] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <kevedo909> los escucho cortados
[01Jun2025 01:43:50.003] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> xd
[01Jun2025 01:43:53.201] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <Santix15> estan unidos en grupos distintos
[01Jun2025 01:43:59.495] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lautiABC> td mal kevin
[01Jun2025 01:43:59.902] [Server thread/WARN] [net.minecraft.world.level.storage.loot.functions.SetItemDamageFunction/]: Couldn't set damage of loot item 0 air
[01Jun2025 01:44:15.247] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <kevedo909> porque
[01Jun2025 01:44:24.895] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <kevedo909> si ustedes son los pajeros de mierda
[01Jun2025 01:45:10.999] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundAcceptTeleportationPacket  id 0 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 01:45:11.000] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133785_": 60621469
}
[01Jun2025 01:45:11.000] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 0 data of 45 extra bytes: 
[01Jun2025 01:45:11.001] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "-1[]]J(`>(7HF320{[*dG"
]
[01Jun2025 01:45:16.560] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lautiABC> gonza
[01Jun2025 01:45:19.898] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lobo3224> que
[01Jun2025 01:45:26.905] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <Santix15> jejeje
[01Jun2025 01:45:31.148] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <Santix15> soooooooooooo
[01Jun2025 01:45:33.000] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <kevedo909> violin
[01Jun2025 01:45:33.547] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lautiABC> pd guntarnos hablar
[01Jun2025 01:45:40.453] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lautiABC> solo hablar
[01Jun2025 01:45:40.453] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lobo3224> noooooooooooo
[01Jun2025 01:45:51.443] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lautiABC> no no quiero pvo
[01Jun2025 01:46:01.051] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <kevedo909> ustedes me quiren dejar todo solo
[01Jun2025 01:46:04.046] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> miedo???
[01Jun2025 01:46:05.909] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> dea
[01Jun2025 01:46:15.205] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <kevedo909> miedo que
[01Jun2025 01:46:22.931] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <kevedo909> concha de tu madre
[01Jun2025 01:46:23.214] [Server thread/WARN] [net.minecraft.world.level.storage.loot.functions.SetItemDamageFunction/]: Couldn't set damage of loot item 0 air
[01Jun2025 01:47:19.570] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lautiABC> gonza
[01Jun2025 01:47:19.719] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: lautiABC was slain by Leech
[01Jun2025 01:47:34.700] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <kevedo909> viste agus, te callas
[01Jun2025 01:47:47.196] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> estaba llegando a la base
[01Jun2025 01:47:50.863] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> putin
[01Jun2025 01:47:52.594] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> dea
[01Jun2025 01:47:53.759] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundAcceptTeleportationPacket  id 0 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 01:47:53.759] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133785_": 111
}
[01Jun2025 01:47:53.759] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 0 data of 48 extra bytes: 
[01Jun2025 01:47:53.760] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "^O7E+&!pAAs320>`(:|4"
]
[01Jun2025 01:48:02.047] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lautiABC> el agus va a ir hablar con vos solo quiero ir as buscar mis cosas y nos vamos
[01Jun2025 01:48:03.249] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <kevedo909> puta tu relacion de poronga
[01Jun2025 01:48:03.371] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Santix15 moved wrongly!
[01Jun2025 01:48:09.573] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <kevedo909> dea
[01Jun2025 01:48:12.199] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <kevedo909> xd
[01Jun2025 01:48:23.757] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> otro mas con gael
[01Jun2025 01:48:24.960] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> xd
[01Jun2025 01:49:46.163] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: kevedo909 was impaled by Drowned
[01Jun2025 01:49:53.403] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> xdsxdxdxdxdxd
[01Jun2025 01:49:57.447] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> dea
[01Jun2025 01:50:17.817] [Server thread/INFO] [net.minecraft.world.entity.npc.Villager/]: Villager Villager['Unemployed'/119453, l='ServerLevel[world]', x=-1462.28, y=-256.00, z=2424.77] died, message: 'Unemployed was killed'
[01Jun2025 01:50:38.848] [Server thread/INFO] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: kevedo909 lost connection: Disconnected
[01Jun2025 01:50:38.848] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: kevedo909 left the game
[01Jun2025 01:50:38.849] [Server thread/INFO] [voicechat/]: [voicechat] Disconnecting client kevedo909
[01Jun2025 01:50:50.500] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> ajaajajajajajja
[01Jun2025 01:50:50.941] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lautiABC> gael 2,0
[01Jun2025 01:52:34.093] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: YOUNGDINO07 fell from a high place
[01Jun2025 01:52:47.760] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lobo3224> te iba a revivir
[01Jun2025 01:53:03.728] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundAcceptTeleportationPacket  id 0 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 01:53:03.728] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133785_": 2912
}
[01Jun2025 01:53:03.728] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 0 data of 47 extra bytes: 
[01Jun2025 01:53:03.729] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "VGq>*296$;320d;#C6]"
]
[01Jun2025 01:55:24.451] [Worker-Main-2/ERROR] [net.minecraft.world.entity.decoration.HangingEntity/]: Hanging entity at invalid position: BlockPos{x=-193, y=-45, z=-211}
[01Jun2025 01:55:24.451] [Worker-Main-2/ERROR] [net.minecraft.world.entity.decoration.HangingEntity/]: Hanging entity at invalid position: BlockPos{x=-183, y=-57, z=-210}
[01Jun2025 01:55:24.451] [Worker-Main-2/ERROR] [net.minecraft.world.entity.decoration.HangingEntity/]: Hanging entity at invalid position: BlockPos{x=-187, y=-49, z=-219}
[01Jun2025 01:55:24.451] [Worker-Main-2/ERROR] [net.minecraft.world.entity.decoration.HangingEntity/]: Hanging entity at invalid position: BlockPos{x=-189, y=-45, z=-211}
[01Jun2025 01:55:24.451] [Worker-Main-2/ERROR] [net.minecraft.world.entity.decoration.HangingEntity/]: Hanging entity at invalid position: BlockPos{x=-191, y=-45, z=-211}
[01Jun2025 01:55:24.457] [Worker-Main-2/ERROR] [net.minecraft.world.entity.decoration.HangingEntity/]: Hanging entity at invalid position: BlockPos{x=-291, y=-57, z=-315}
[01Jun2025 01:55:24.457] [Worker-Main-2/ERROR] [net.minecraft.world.entity.decoration.HangingEntity/]: Hanging entity at invalid position: BlockPos{x=-292, y=-57, z=-313}
[01Jun2025 01:55:24.457] [Worker-Main-2/ERROR] [net.minecraft.world.entity.decoration.HangingEntity/]: Hanging entity at invalid position: BlockPos{x=-294, y=-57, z=-313}
[01Jun2025 01:55:24.457] [Worker-Main-2/ERROR] [net.minecraft.world.entity.decoration.HangingEntity/]: Hanging entity at invalid position: BlockPos{x=-296, y=-57, z=-313}
[01Jun2025 01:55:24.457] [Worker-Main-2/ERROR] [net.minecraft.world.entity.decoration.HangingEntity/]: Hanging entity at invalid position: BlockPos{x=-295, y=-57, z=-315}
[01Jun2025 01:55:24.458] [Worker-Main-2/ERROR] [net.minecraft.world.entity.decoration.HangingEntity/]: Hanging entity at invalid position: BlockPos{x=-293, y=-57, z=-315}
[01Jun2025 01:55:24.469] [Worker-Main-2/ERROR] [net.minecraft.world.entity.decoration.HangingEntity/]: Hanging entity at invalid position: BlockPos{x=-183, y=-57, z=-201}
[01Jun2025 01:55:24.469] [Worker-Main-2/ERROR] [net.minecraft.world.entity.decoration.HangingEntity/]: Hanging entity at invalid position: BlockPos{x=-183, y=-58, z=-205}
[01Jun2025 01:56:05.396] [Worker-Main-2/WARN] [net.minecraft.world.level.levelgen.structure.pools.JigsawPlacement/]: Empty or non-existent pool: minecraft:
[01Jun2025 01:56:55.670] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Player 71cf154b-6e54-326e-9700-98d60017873d timed out
[01Jun2025 01:56:55.670] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Reconnecting player lautiABC
[01Jun2025 01:56:55.671] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Sent secret to lautiABC
[01Jun2025 01:57:19.911] [Server thread/WARN] [net.minecraft.network.syncher.SynchedEntityData/]: defineId called for: class com.cerbon.bosses_of_mass_destruction.entity.util.BaseEntity from class com.cerbon.bosses_of_mass_destruction.entity.custom.void_blossom.hitbox.NetworkedHitboxManager
[01Jun2025 01:57:34.001] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Successfully authenticated player 71cf154b-6e54-326e-9700-98d60017873d
[01Jun2025 01:57:35.083] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Successfully validated connection of player 71cf154b-6e54-326e-9700-98d60017873d
[01Jun2025 01:57:35.083] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Player lautiABC (71cf154b-6e54-326e-9700-98d60017873d) successfully connected to voice chat
[01Jun2025 01:59:24.544] [Server thread/WARN] [net.minecraft.world.level.storage.loot.functions.SetItemDamageFunction/]: Couldn't set damage of loot item 0 air
[01Jun2025 02:00:15.761] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Santix15 has made the advancement [Wax Off]
[01Jun2025 02:00:57.765] [Netty Epoll Server IO #3/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundAcceptTeleportationPacket  id 0 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 02:00:57.766] [Netty Epoll Server IO #3/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133785_": 1675631
}
[01Jun2025 02:00:57.766] [Netty Epoll Server IO #3/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 0 data of 46 extra bytes: 
[01Jun2025 02:00:57.766] [Netty Epoll Server IO #3/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "Yr2G^6D(!h:r320}f)qS5i"
]
[01Jun2025 02:02:05.152] [Server thread/INFO] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Santix15 lost connection: Disconnected
[01Jun2025 02:02:05.152] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Santix15 left the game
[01Jun2025 02:02:05.152] [Server thread/INFO] [voicechat/]: [voicechat] Disconnecting client Santix15
[01Jun2025 02:02:05.384] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lautiABC> gonza
[01Jun2025 02:02:12.237] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lobo3224> que
[01Jun2025 02:02:29.385] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lautiABC> piuedo ir a buscar mi cosas y me puedo ir
[01Jun2025 02:02:52.706] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lobo3224> le di un stack de hierro a agus
[01Jun2025 02:03:02.589] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lautiABC> o
[01Jun2025 02:03:09.798] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> confirmo
[01Jun2025 02:03:12.141] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lautiABC> busco mis cosas y nos vamos
[01Jun2025 02:03:22.495] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lobo3224> es que pasaron cosas
[01Jun2025 02:03:45.797] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lautiABC> q ps
[01Jun2025 02:04:36.755] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lobo3224> te doy un stak de hierro
[01Jun2025 02:04:52.555] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> stack
[01Jun2025 02:04:56.789] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lobo3224> eso
[01Jun2025 02:04:59.365] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundAcceptTeleportationPacket  id 0 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 02:04:59.366] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133785_": 119
}
[01Jun2025 02:04:59.366] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 0 data of 48 extra bytes: 
[01Jun2025 02:04:59.366] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "fA~{WopmB+M*&(S3203q]/!a4L"
]
[01Jun2025 02:05:01.488] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <YOUNGDINO07> xd
[01Jun2025 02:06:38.787] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lobo3224> sali del grupo
[01Jun2025 02:07:00.586] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Player 71cf154b-6e54-326e-9700-98d60017873d timed out
[01Jun2025 02:07:00.587] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Reconnecting player lautiABC
[01Jun2025 02:07:00.587] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Sent secret to lautiABC
[01Jun2025 02:07:42.428] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Successfully authenticated player 71cf154b-6e54-326e-9700-98d60017873d
[01Jun2025 02:07:43.533] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Successfully validated connection of player 71cf154b-6e54-326e-9700-98d60017873d
[01Jun2025 02:07:43.533] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Player lautiABC (71cf154b-6e54-326e-9700-98d60017873d) successfully connected to voice chat
[01Jun2025 02:08:26.384] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lautiABC> ma vas comida
[01Jun2025 02:08:56.427] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lautiABC> muchas gracias q tengas una bonita nocha
[01Jun2025 02:09:55.206] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 02:09:55.206] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=860, z=1073200}"
}
[01Jun2025 02:09:55.206] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 02:09:55.206] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[70, 29, 39, 51, 97, 65, -34, 91, 83, -36, -115]"
]
[01Jun2025 02:09:56.638] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 02:09:56.639] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=812, z=1073200}"
}
[01Jun2025 02:09:56.639] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 02:09:56.639] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[-46, -21, 25, -3, 99, -16, 0, -100, 8, 48, 92]"
]
[01Jun2025 02:09:56.837] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 02:09:56.837] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=5768192, y=962, z=1122352}"
}
[01Jun2025 02:09:56.838] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 02:09:56.838] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[-57, 109, 88, -119, -36, -25, -17, -52, 12, -44, -91]"
]
[01Jun2025 02:09:57.081] [Netty Epoll Server IO #3/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 02:09:57.082] [Netty Epoll Server IO #3/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 0,
  "f_133799_": "BlockPos{x=-8125440, y=947, z=905264}"
}
[01Jun2025 02:09:57.083] [Netty Epoll Server IO #3/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 02:09:57.083] [Netty Epoll Server IO #3/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[25, -45, -103, -30, -64, 90, -98, -1, 79, 60, 116]"
]
[01Jun2025 02:09:57.181] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 02:09:57.182] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=883, z=1073200}"
}
[01Jun2025 02:09:57.182] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 02:09:57.182] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[4, -17, -109, -88, -123, -5, -2, -83, 26, 39, -8]"
]
[01Jun2025 02:09:57.361] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 02:09:57.362] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 0,
  "f_133799_": "BlockPos{x=-11271168, y=792, z=856112}"
}
[01Jun2025 02:09:57.362] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 02:09:57.362] [Netty Epoll Server IO #6/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[38, -49, 74, -76, 36, 108, 90, 56, -24, 105, -94]"
]
[01Jun2025 02:09:57.508] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 02:09:57.508] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=806, z=1073200}"
}
[01Jun2025 02:09:57.508] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 02:09:57.509] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[90, -88, 90, 43, 19, -84, 12, 56, -47, 72, -118]"
]
[01Jun2025 02:09:57.536] [Netty Epoll Server IO #0/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 02:09:57.536] [Netty Epoll Server IO #0/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=842, z=1073200}"
}
[01Jun2025 02:09:57.536] [Netty Epoll Server IO #0/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 02:09:57.536] [Netty Epoll Server IO #0/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[-91, 40, -89, -55, 116, 117, -7, 20, 37, 74, -87]"
]
[01Jun2025 02:09:57.542] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 02:09:57.542] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=860, z=1073200}"
}
[01Jun2025 02:09:57.542] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 02:09:57.542] [Netty Epoll Server IO #1/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[-23, 122, -54, -109, -11, 124, -14, 77, -112, -123, 21]"
]
[01Jun2025 02:09:58.310] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 02:09:58.310] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=873, z=1073200}"
}
[01Jun2025 02:09:58.310] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 02:09:58.311] [Netty Epoll Server IO #2/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[59, -96, -14, -108, -39, -126, -61, -112, 9, 75, -115]"
]
[01Jun2025 02:09:58.670] [Netty Epoll Server IO #3/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 02:09:58.671] [Netty Epoll Server IO #3/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=970, z=1073200}"
}
[01Jun2025 02:09:58.671] [Netty Epoll Server IO #3/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 02:09:58.671] [Netty Epoll Server IO #3/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[65, -83, 125, -27, -107, 66, 47, 126, 94, -38, 83]"
]
[01Jun2025 02:09:58.934] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 02:09:58.934] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=976, z=1073200}"
}
[01Jun2025 02:09:58.934] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 02:09:58.934] [Netty Epoll Server IO #4/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[-80, 69, -42, 44, 90, -83, 74, -110, 26, 71, 66]"
]
[01Jun2025 02:09:58.962] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundBlockEntityTagQuery  id 1 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 02:09:58.962] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133798_": 1,
  "f_133799_": "BlockPos{x=2622464, y=973, z=1073200}"
}
[01Jun2025 02:09:58.962] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 1 data of 11 extra bytes: 
[01Jun2025 02:09:58.963] [Netty Epoll Server IO #5/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "<partial numeric data>",
  "[66, 17, 9, 10, -30, 73, -49, -120, -67, 86, 6]"
]
[01Jun2025 02:13:44.391] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lautiABC> q ps agus
[01Jun2025 02:13:52.977] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <lautiABC> ?????
[01Jun2025 02:16:58.126] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: YOUNGDINO07 has made the advancement [Bypassing the Expiration Date]
[01Jun2025 02:17:49.427] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Player 71cf154b-6e54-326e-9700-98d60017873d timed out
[01Jun2025 02:17:49.427] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Reconnecting player lautiABC
[01Jun2025 02:17:49.427] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Sent secret to lautiABC
[01Jun2025 02:18:20.361] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Successfully authenticated player 71cf154b-6e54-326e-9700-98d60017873d
[01Jun2025 02:18:21.422] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Successfully validated connection of player 71cf154b-6e54-326e-9700-98d60017873d
[01Jun2025 02:18:21.423] [VoiceChatPacketProcessingThread/INFO] [voicechat/]: [voicechat] Player lautiABC (71cf154b-6e54-326e-9700-98d60017873d) successfully connected to voice chat
[01Jun2025 02:19:28.334] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:ServerboundAcceptTeleportationPacket  id 0 larger than expected error detected, printing packet and buffer. Stacktrace gets logged after this
[01Jun2025 02:19:28.335] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
{
  "f_133785_": 105
}
[01Jun2025 02:19:28.335] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet:PooledUnsafeDirectByteBuf  id 0 data of 48 extra bytes: 
[01Jun2025 02:19:28.335] [Netty Epoll Server IO #7/WARN] [com.connectivity.Connectivity/]: Packet data:
[
  "3`~IJ5xJ4\"idQ320LhdR"
]
[01Jun2025 02:21:12.875] [Server thread/WARN] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: YOUNGDINO07 moved wrongly!
